scripts/docker-compose.yml

```diff
@@ -88,7 +88,7 @@ services:
     ports:
       - "15432:5432"
   db-host-inventory:
-    image: postgres
+    image: debezium/postgres
```

Comment:
@patilsuraj767  Do we need postgres from debezium? I might be missing the context here.

Reply:
Yes, we need it. To use kafka connector with postgres there is some specific postgres setting needed to be enabled. `debezium/postgres` image has those setting enabled.